File: Data Structures & Algorithms/find-duplicate-integer/submission-0.cs
public class Solution {
    public int FindDuplicate(int[] nums) {
        foreach (var num in nums) {
            if (nums[Math.Abs(num) - 1] < 0) return Math.Abs(num);
            nums[Math.Abs(num) - 1] *= -1;
        }
        return 0;
    }
}

File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        i = 0
        j = 0
        n = len(nums)
        while j < n:
            while j < n and nums[i] == nums[j]:
                j += 1
            i += 1
            if j < n:
                nums[i] = nums[j]
        return i